proto/dymensionxyz/dymension/dymns/tx.proto

```diff
@@ -0,0 +1,284 @@
+syntax = "proto3";
+package dymensionxyz.dymension.dymns;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/msg/v1/msg.proto";
+import "dymensionxyz/dymension/dymns/market.proto";
+
+option go_package = "github.com/dymensionxyz/dymension/v3/x/dymns/types";
+
+// Msg defines the Msg service.
+service Msg {
+    // RegisterName is message handler, handles registration of a new Dym-Name
+    // or extends the ownership duration of an existing Dym-Name.
+    rpc RegisterName(MsgRegisterName) returns (MsgRegisterNameResponse) {}
+    // TransferOwnership is message handler,
+    // handles transfer of ownership of a Dym-Name, performed by the owner.
+    rpc TransferOwnership(MsgTransferOwnership) returns (MsgTransferOwnershipResponse) {}
+    // SetController is message handler,
+    // handles setting a controller for a Dym-Name, performed by the owner.
+    rpc SetController(MsgSetController) returns (MsgSetControllerResponse) {}
+    // UpdateResolveAddress is message handler,
+    // handles updating Dym-Name-Address resolution configuration, performed by the controller.
+    rpc UpdateResolveAddress(MsgUpdateResolveAddress) returns (MsgUpdateResolveAddressResponse) {}
+    // UpdateDetails is message handler,
+    // handles updating Dym-Name details, performed by the controller.
+    rpc UpdateDetails(MsgUpdateDetails) returns (MsgUpdateDetailsResponse) {}
+
+    // PlaceSellOrder is message handler,
+    // handles creating a Sell-Order that advertise a Dym-Name/Alias is for sale, performed by the owner.
+    rpc PlaceSellOrder(MsgPlaceSellOrder) returns (MsgPlaceSellOrderResponse) {}
```

Comment:
Sell order can be deleted when no one placed a bid on it.
If someone placed a bid on it so it can not be cancelled.
| State\Change | Lower price | Increase price |
| --- | --- | --- |
| No Bid | Cancel and re-place | Cancel and re-place | 
| Bid | Not make sense one someone placed a price that >= min price placed by owner | That's un-acceptable when someone placed a bid and you want increase price |

If someone see the asset is open for sale but the price is too high, they can actually send an offer (Buy-Order) for negotiation.

So imo we don't need to update Sell-Order.